src/main/java/reactor/netty/http/client/HttpClient.java

```diff
@@ -657,7 +657,22 @@ public final HttpClient doAfterResponse(BiConsumer<? super HttpClientResponse, ?
 	 */
 	public final HttpClient doAfterResponseSuccess(BiConsumer<? super HttpClientResponse, ? super Connection> doAfterResponseSuccess) {
 		Objects.requireNonNull(doAfterResponseSuccess, "doAfterResponseSuccess");
-		return new HttpClientDoOn(this, null, null, null, null, doAfterResponseSuccess);
+		return new HttpClientDoOn(this, null, null, null, null, doAfterResponseSuccess, null);
+	}
+
+	/**
+	 * Setup a callback called when {@link HttpClientResponse} failed with
+	 * {@link reactor.netty.channel.AbortedException#isConnectionReset(Throwable)}
+	 * and {@link HttpClientState#RETRY_ENABLED} has been emitted.
```

Comment:
```suggestion
	 * Setup a callback called when {@link HttpClientResponse} failed with
	 * {@link reactor.netty.channel.AbortedException#isConnectionReset(Throwable)}
	 * and a retry to connect is about to be performed.
```

Reply:
done, method name changed to `doOnRetry`